Inventory Management System/UserInterface.cs

```diff
@@ -10,7 +10,70 @@ internal class UserInterface
     {
         static void Main(string[] args)
         {
+            
 
+            while (true)
+            {
+                Console.WriteLine("Welcome to the Inventory Management System!");
+                DisplayMenu();
+                int choice = GetChoiceFromUser();
```

Comment:
don't use explicate variable types, please use "var", in case of null you can use, e.x: (int?) null;